bin/build-boilerplate.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Usage like so:
+# ./build-boilerplate.sh
+
+FILE='boilerplate.zip'
+
+# This allows us to avoid duplicating files in git
+FILES_TO_COPY='zapierwrapper.js'
+
+if [ -f $FILE ]
+then
+   rm -f $FILE
+fi
+
+# Copy files
+cp include/$FILES_TO_COPY boilerplate/
```

Comment:
I'm rusty on bash, but I believe it's best practice to wrap expansions in paths in quotes: `cp "include/$FILES_TO_COPY" boilerplate/`. That way, if we add more files to copy later, we won't worry about weird path expansions other places in the file